tests/ci-helper.test.ts

```diff
@@ -158,6 +163,75 @@ async function setupRepos(instance: string):
     return { worktree, gggLocal, gggRemote };
 }
 
+/**
+ * Connect to imap mail server.  Opens the INBOX as the current folder.
+ */
+async function imapConnect(): Promise <ImapSimple> {
+    const { smtpUser, smtpPass, imapHost } = await getEmailInfo();
+
+    const config = {
+        imap: {
+            user: smtpUser,
+            password: smtpPass,
+            host: imapHost,
+            port: 993,
+            tls: true,
+            authTimeout: 3000
+        }
+    };
+
+    const connection = await connect(config);
+    await connection.openBox("INBOX");
+
+    return connection;
+}
+
+/**
+ * Terminate the imap mail server connection.
+ *
+ * @param connection
+ */
+async function imapDisConnect(connection: ImapSimple): Promise <void> {
```

Comment:
```suggestion
async function imapDisconnect(connection: ImapSimple): Promise <void> {
```

Reply:
Sorry, not what I would have done from scratch.  Changed.